pypeit/tests/test_traceslits.py

```diff
@@ -24,42 +24,47 @@ def chk_for_files(root):
 
 @dev_suite_required
 def test_addrm_slit():
+    # TODO This TEST needs to be rewritten becuase from_master_files is defunct. THe master only includes
```

Comment:
Unlike the DEV_SUITE, these tests check performance.
Am going to turn them back on.

Reply:
Okay but they need to be updated because we are not able to reinstantiate the class from a file.